unskript-ctl/unskript_ctl_database.py

```diff
@@ -317,6 +317,24 @@ def get_check_by_name(self, check_name: str):
                 snippet.get('metadata').get('action_is_check') and
                 snippet.get('metadata').get('action_entry_function') == check_name]
     
+    def get_info_actions(self):
+        """This routine returns the actions that has CATEGORY_TYPE_INFORMATION in action_category"""
```

Comment:
Do we have a PR for adding the TYPE_INFORMATION to the info legos?

Reply:
https://github.com/unskript/Awesome-CloudOps-Automation/pull/960